analyses/molecular-subtyping-EWS/01-reclassify_as_ewings.Rmd

```diff
@@ -79,9 +110,9 @@ clinical_wgs <- clinical %>%
          Kids_First_Participant_ID, 
          sample_id)
 
-# this drops normal samples
-clinical <- clinical_rnaseq %>% 
-  inner_join(clinical_wgs,
+# Join together
+clinical_reclassified <- clinical_rnaseq %>% 
```

Comment:
should we remove the sample_type column in final output file since they are all Tumor now?

Reply:
Yes, good idea!